ajax/libs/winstrap/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "winstrap",
+  "filename": "css/winstrap.min.css",
+  "description": "Winstrap is the official Bootstrap theme for Microsoft design language.",
+  "version": "0.5.11",
+  "author": "Microsoft",
+  "license": "MIT",
```

Comment:
I think we also need to add `OFL-1.1` for font license. https://github.com/winjs/winstrap/blob/4792deb8f365a64b25bcb110252848cc475717a6/dist/fonts/LICENSE.txt